test/format/format_util.c

```diff
@@ -386,13 +386,14 @@ atou32(const char *tag, const char *s, int match)
  *     Open a WiredTiger session.
  */
 void
-wt_wrap_open_session(WT_CONNECTION *conn, SAP *sap, const char *track, WT_SESSION **sessionp)
+wt_wrap_open_session(
+  WT_CONNECTION *conn, SAP *sap, const char *track, const char *cfg, WT_SESSION **sessionp)
 {
     WT_SESSION *session;
 
     *sessionp = NULL;
 
-    testutil_check(conn->open_session(conn, NULL, NULL, &session));
+    testutil_check(conn->open_session(conn, NULL, cfg, &session));
 
     if (g.trace_conn != NULL && sap->trace == NULL)
         testutil_check(g.trace_conn->open_session(g.trace_conn, NULL, NULL, &sap->trace));
```

Comment:
@sueloverso, do I need to pass the same session configuration here as well?

Reply:
No, this is the session just for calling `log_printf` on the operations. There is no prefetching or any other special thing we do in that other connection. There is no data written.